integration/jnifuse/native/src/main/native/libjnifuse/jnifuse_impls.cc

```diff
@@ -11,9 +11,145 @@
 
 #include "jnifuse_impls.h"
 
+#include <fuse.h>
+
 #include "debug.h"
 #include "jnifuse_fs.h"
 
+#if FUSE_USE_VERSION >= 30
+
+int chmod_wrapper(const char *path, mode_t mode, struct fuse_file_info *fi) {
```

Comment:
Please refer to other fuse3 implementation to see how the fi is used?

Here it is just ignored, it might be used to optimize something.

Reply:
It is not necessary to use it at this time, but will provide extra performance benefit.